bin/citrea/src/main.rs

```diff
@@ -72,20 +72,30 @@ async fn main() -> Result<(), anyhow::Error> {
     };
     initialize_logging(logging_level);
 
-    let rollup_config_path = args.rollup_config_path.as_str();
+    let rollup_config_path = args.rollup_config_path.unwrap();
```

Comment:
i think no unwrap here, if no --rollup-config-path is passed, try to do from_env() directly, if that fails, regardless of launch mode, return Err

so rollup config path is Option<String> and not forced as an argument.

Reply:
got it